README.md

```diff
@@ -1,5 +1,89 @@
-# pagination-node [![Build Status](https://travis-ci.com/Wolox/pagination-node.svg?branch=master)](https://travis-ci.com/Wolox/pagination-node)
-Pagination npm package for an API Node app. Parameters of pagination, custom output format, custom configuration, adaptable to different kind of entities to paginate.
+# Pagination-Node [![Build Status](https://travis-ci.com/Wolox/pagination-node.svg?branch=master)](https://travis-ci.com/Wolox/pagination-node)
+
+Pagination Format NPM package for collections/entities.
+
+
+## Summary
+The package allows you to transform a collection of objects into a paginated JSON response format with certain pagination parameters.
+
+The package is able to slice the entire collection sent based on the `limit` and the `page` parameters that can be specified to the `paginate` method of the package, and therefore return just the portion of the collection needed, as a page, along with the related pagination params.
```

Comment:
As to avoid repetition
"It's able to..."
"Pagination-Node slices the entire...."